Libraries/Hotcakes.Commerce/app.config

```diff
@@ -23,6 +23,10 @@
         <assemblyIdentity name="MiniProfiler" publicKeyToken="b44f9351044011a3" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-3.1.1.140" newVersion="3.1.1.140" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
```

Comment:
There are multiple references to `Newtonsoft.Json` in this update, and the versions don't match.  We need to prevent compile-time warnings and potential exceptions/debugging issues later.  Please be sure all identical references reflect the same reference version.

Reply:
Fixed.